File: Exemplos SmartPOS/App_IntentDigitalHUB_Java/Elgin_IntentDigitalHUB_SmartPOS_Java-font/app/src/main/java/com/elgin/java_intentdigitalhub_smartpos/MainActivity.java
package com.elgin.java_intentdigitalhub_smartpos;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.LinearLayout;

import com.elgin.java_intentdigitalhub_smartpos.Activities.ActivityUtils;
import com.elgin.java_intentdigitalhub_smartpos.Activities.BarCodeReader.BarCodeReaderActivity;
import com.elgin.java_intentdigitalhub_smartpos.Activities.ElginPay.ElginPayActivity;
import com.elgin.java_intentdigitalhub_smartpos.Activities.Printer.PrinterMenuActivity;

public class MainActivity extends AppCompatActivity {

    private LinearLayout buttonElginPay;
    private LinearLayout buttonPrinterMenu;
    private LinearLayout buttonBarCodeReader;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        buttonElginPay = findViewById(R.id.buttonElginPay);
        buttonPrinterMenu = findViewById(R.id.buttonPrinterMenu);
        buttonBarCodeReader = findViewById(R.id.buttonBarCodeReader);

        buttonElginPay.setOnClickListener(v -> ActivityUtils.startNewActivity(this, ElginPayActivity.class));

        buttonPrinterMenu.setOnClickListener(v -> ActivityUtils.startNewActivity(this, PrinterMenuActivity.class));

        buttonBarCodeReader.setOnClickListener(v -> ActivityUtils.startNewActivity(this, BarCodeReaderActivity.class));
    }
}

File: Exemplos SmartPOS/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_SmartPOS_Kotlin-font/app/src/main/java/com/elgin/kotlin_intent_digital_hub_smartpos/MainActivity.kt
package com.elgin.kotlin_intent_digital_hub_smartpos

import ActivityUtils.startNewActivity
import android.os.Bundle
import android.view.View
import android.widget.LinearLayout
import androidx.appcompat.app.AppCompatActivity
import com.elgin.kotlin_intent_digital_hub_smartpos.Activities.BarCodeReader.BarCodeReaderActivity
import com.elgin.kotlin_intent_digital_hub_smartpos.Activities.ElginPay.ElginPayActivity
import com.elgin.kotlin_intent_digital_hub_smartpos.Activities.Printer.PrinterMenuActivity

open class MainActivity : AppCompatActivity() {
    private var buttonElginPay: LinearLayout? = null
    private var buttonPrinterMenu: LinearLayout? = null
    private var buttonBarCodeReader: LinearLayout? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        buttonElginPay = findViewById<LinearLayout>(R.id.buttonElginPay)
        buttonPrinterMenu = findViewById<LinearLayout>(R.id.buttonPrinterMenu)
        buttonBarCodeReader = findViewById<LinearLayout>(R.id.buttonBarCodeReader)
        buttonElginPay!!.setOnClickListener { v: View? ->
            startNewActivity(
                this,
                ElginPayActivity::class.java
            )
        }
        buttonPrinterMenu!!.setOnClickListener { v: View? ->
            startNewActivity(
                this,
                PrinterMenuActivity::class.java
            )
        }
        buttonBarCodeReader!!.setOnClickListener { v: View? ->
            startNewActivity(
                this,
                BarCodeReaderActivity::class.java
            )
        }
    }
}